### ISSUE TITLE
After upgrade to 3.17.0 error: Action data is missing or empty. Action: app_site_editor_template_types

### ISSUE THREAD

**Issue by luigi23n** [elementor/elementor#24103]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

After upgrade to Elementor 3.17.0 and go to Theme bulider display error: Action data is missing or empty. Action: app_site_editor_template_types. On Elementor 3.16.6 in same confuguration of website this error not show and everything work great.

### Steps to reproduce

After upgrade to Elementor 3.17.0 and go to Theme bulider 

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

== Server Environment ==
	Operating System: Linux
	Software: LiteSpeed
	MySQL version: MariaDB Server v10.6.15
	PHP Version: 8.1.24
	PHP Memory Limit: 1000M
	PHP Max Input Vars: 100000
	PHP Max Post Size: 500M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.3.2
	Site URL: xxx
	Home URL: xxx
	WP Multisite: No
	Max Upload Size: 500 MB
	Memory limit: 40M
	Max Memory limit: 1000M
	Permalink Structure: /%postname%/
	Language: pl-PL
	Timezone: Europe/Warsaw
	Admin Email: xxx
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.8.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: pl_PL
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:109.0) Gecko/20100101 Firefox/118.0

== Active Plugins ==
	Cookie Notice &amp; Compliance for GDPR / CCPA
		Version: 2.4.11.1
		Author: Hu-manity.co

	Elementor
		Version: 3.17.0
		Author: Elementor.com

	LiteSpeed Cache
		Version: 5.7
		Author: LiteSpeed Technologies

	PRO Elements
		Version: 3.16.2
		Author: PROElements.org

	WPML Multilingual CMS
		Version: 4.6.6
		Author: OnTheGoSystems

	WPML String Translation
		Version: 3.2.8
		Author: OnTheGoSystems

	Yoast SEO
		Version: 21.4
		Author: Team Yoast

== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	

== Eksperymenty z Elementorem ==
	Zoptymalizowany wynik DOM: Domyślnie aktywne
	Ulepszone ładowanie zasobów: Domyślnie aktywne
	Poprawione ładowanie CSS: Domyślnie aktywne
	Czcionka ikon: Domyślnie nieaktywne
	Dodatkowe niestandardowe punkty zmiany: Domyślnie aktywne
	admin_menu_rearrangement: Domyślnie nieaktywne
	Flexbox Container: Domyślnie nieaktywne
	Upgrade Swiper Library: Domyślnie aktywne
	Grid Container: Domyślnie nieaktywne
	Default to New Theme Builder: Domyślnie aktywne
	Nagłówek i stopka motywu Hello: Domyślnie aktywne
	Editor Top Bar: Domyślnie nieaktywne
	Optimized Gutenberg Loading: Domyślnie nieaktywne
	Build with AI: Domyślnie nieaktywne
	Strony docelowe: Domyślnie aktywne
	Elementy zagnieżdżone: Domyślnie nieaktywne
	Leniwe ładowanie obrazów tła: Domyślnie nieaktywne
	Optimize Image Loading: Domyślnie nieaktywne
	Global Style Guide: Domyślnie aktywne
	Page Transitions: Domyślnie aktywne
	Notes: Domyślnie aktywne
	Form Submissions: Domyślnie aktywne
	Scroll Snap: Domyślnie aktywne
	Menu: Domyślnie nieaktywne
	Taxonomy Filter: Domyślnie nieaktywne

== Dziennik ==
	
Log: showing 20 of 372023-09-27 09:32:58 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.15.3',
  'to' => '3.16.4',
)]
2023-10-09 15:05:09 [info] elementor::elementor_updater Started 
2023-10-09 15:05:09 [info] Elementor/Upgrades - _on_each_version Start  
2023-10-09 15:05:09 [info] Elementor/Upgrades - _on_each_version Finished 
2023-10-09 15:05:09 [info] Elementor/Upgrades - _v_3_16_0_container_updates Start  
2023-10-09 15:05:09 [info] Elementor/Upgrades - _v_3_16_0_container_updates Finished 
2023-10-09 15:05:09 [info] Elementor/Upgrades - _v_3_17_0_site_settings_updates Start  
2023-10-09 15:05:09 [info] Elementor/Upgrades - _v_3_17_0_site_settings_updates Finished 
2023-10-09 15:05:09 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.16.4',
  'to' => '3.16.5',
)]
2023-10-09 15:05:15 [info] elementor-pro::elementor_pro_updater Started 
2023-10-09 15:05:15 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-10-09 15:05:15 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-10-09 15:05:16 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.15.1',
  'to' => '3.16.2',
)]
2023-10-25 14:20:10 [info] elementor::elementor_updater Started 
2023-10-25 14:20:10 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.16.5',
  'to' => '3.17.0',
)]
2023-10-25 14:20:10 [info] Elementor/Upgrades - _on_each_version Start  
2023-10-25 14:20:10 [info] Elementor/Upgrades - _on_each_version Finished 
2023-10-25 14:20:10 [info] Elementor/Upgrades - _v_3_17_0_site_settings_updates Start  
2023-10-25 14:20:10 [info] Elementor/Upgrades - _v_3_17_0_site_settings_updates Finished 
2023-10-25 14:20:10 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.16.5',
  'to' => '3.17.0',
)]

== Elementor - Compatibility Tag ==
	

== Elementor Pro - Compatibility Tag ==

**Comment by bgmastermind** [comment]
I have exactly the same issue. I've tried disabling "Connect Polylang for Elementor" and some other plugins, but it didn't help.
					
```
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: mariadb.org binary distribution v10.5.10
	PHP Version: 8.1.18
	PHP Memory Limit: 1024M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 256M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.3.2
	Site URL: https://xxx
	Home URL: https://xxx
	WP Multisite: No
	Max Upload Size: 256 MB
	Memory limit: 40M
	Max Memory limit: 1024M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: 2
	Admin Email: 
	Debug Mode: Inactive

== Theme ==
	Name: Astra Child
	Version: 1.0.0
	Author: xxxx
	Child Theme: Yes
	Parent Theme Name: Astra
	Parent Theme Version: 4.4.0
	Parent Theme Author: Brainstorm Force

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/118.0.0.0 Safari/537.36

== Active Plugins ==
	Companion Auto Update
		Version: 3.8.7.1
		Author: Papin Schipper

	Complianz | GDPR/CCPA Cookie Consent
		Version: 6.5.5
		Author: Really Simple Plugins

	Connect Polylang for Elementor
		Version: 2.4.3
		Author: Creame

	Duplicate Page
		Version: 4.5.3
		Author: mndpsingh287

	Elementor
		Version: 3.17.0
		Author: Elementor.com

	Elementor Pro
		Version: 3.16.2
		Author: Elementor.com

	Import / Export Customizer Settings
		Version: 1.0.7
		Author: Brainstorm Force

	Optimize Database after Deleting Revisions
		Version: 5.1
		Author: CAGE Web Design | Rolf van Gelder, Eindhoven, The Netherlands

	Polylang
		Version: 3.5.2
		Author: WP SYNTEX

	Rank Math SEO
		Version: 1.0.204
		Author: Rank Math

	Redis Object Cache
		Version: 2.4.4
		Author: Till Krüss

	Spectra
		Version: 2.8.0
		Author: Brainstorm Force

	Starter Templates
		Version: 3.4.4
		Author: Brainstorm Force

	UpdraftPlus - Backup/Restore
		Version: 1.23.10
		Author: UpdraftPlus.Com, DavidAnderson

	WebP Express
		Version: 0.25.8
		Author: Bjørn Rosell

	WPForms Lite
		Version: 1.8.4.1
		Author: WPForms

	WP Mail SMTP
		Version: 3.9.0
		Author: WP Mail SMTP

	WP Sitemap Page
		Version: 1.9.2
		Author: Tony Archambeau

	WP Super Cache
		Version: 1.10.0
		Author: Automattic


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	


== Elementor Experiments ==
	Optimized DOM Output: Active by default
	Improved Asset Loading: Active by default
	Improved CSS Loading: Active by default
	Inline Font Icons: Inactive by default
	Additional Custom Breakpoints: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Active by default
	Grid Container: Active
	Default to New Theme Builder: Active by default
	Editor Top Bar: Active
	Optimized Gutenberg Loading: Inactive by default
	Build with AI: Inactive by default
	Landing Pages: Active by default
	Nested Elements: Active
	Lazy Load Background Images: Inactive
	Optimize Image Loading: Inactive by default
	Pages Panel: Inactive by default
	Global Style Guide: Inactive by default
	Page Transitions: Active by default
	Notes: Active by default
	Form Submissions: Active by default
	Scroll Snap: Active by default
	Menu: Inactive by default
	Taxonomy Filter: Inactive by default


== Log ==
	
PHP: 2023-09-25 14:21:08 [warning X 2][/home/web/xxxxx.com/public/wp-content/plugins/elementor/includes/managers/image.php::113] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-09-26 08:00:17 [notice X 9][/home/web/xxxxx.com/public/wp-content/plugins/elementor/includes/utils.php::727] strtolower(): Passing null to parameter #1 ($string) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-10-16 22:56:31 [error X 1][/home/web/xxxxx.com/public/wp-content/plugins/elementor/data/v2/base/controller.php::227] Uncaught Elementor\Data\V2\Base\Exceptions\WP_Error_Exception: Method 'WP_REST_Controller::get_item_permissions_check' not implemented. Must be overridden in subclass. in /home/web/xxxxx.com/public/wp-content/plugins/elementor/data/v2/base/controller.php:227
Stack trace:
#0 /home/web/xxxxx.com/public/wp-content/plugins/elementor/data/v2/base/base-route.php(105): Elementor\Data\V2\Base\Controller->get_permission_callback()
#1 /home/web/xxxxx.com/public/wp-content/plugins/elementor/data/v2/base/base-route.php(242): Elementor\Data\V2\Base\Base_Route->get_permission_callback()
#2 /home/web/xxxxx.com/public/wp-includes/rest-api/class-wp-rest-server.php(1161): Elementor\Data\V2\Base\Base_Route->Elementor\Data\V2\Base\{closure}()
#3 /home/web/xxxxx.com/public/wp-includes/rest-api/class-wp-rest-server.php(1041): WP_REST_Server->respond_to_request()
#4 /home/web/xxxxx.com/public/wp-includes/rest-api/class-wp-rest-server.php(431): WP_REST_Server->dispatch()
#5 /home/web/xxxxx.com/public/wp-includes/rest-api.php(418): WP_REST_Server->serve_request()
#6 /home/web/xxxxx.com/public/wp-includes/class-wp-hook.php(310): rest_api_loaded()
#7 /home/web/xxxxx.com/public/wp-includes/class-wp-hook.php(334): WP_Hook->apply_filters()
#8 /home/web/xxxxx.com/public/wp-includes/plugin.php(565): WP_Hook->do_action()
#9 /home/web/xxxxx.com/public/wp-includes/class-wp.php(398): do_action_ref_array()
#10 /home/web/xxxxx.com/public/wp-includes/class-wp.php(779): WP->parse_request()
#11 /home/web/xxxxx.com/public/wp-includes/functions.php(1335): WP->main()
#12 /home/web/xxxxx.com/public/wp-blog-header.php(16): wp()
#13 /home/web/xxxxx.com/public/index.php(17): require('...')
#14 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

Log: showing 20 of 322023-10-16 08:56:39 [info] Elementor/Upgrades - _on_each_version Start  
2023-10-16 08:56:39 [info] Elementor/Upgrades - _on_each_version Finished 
2023-10-16 08:56:39 [info] Elementor/Upgrades - _v_3_17_0_site_settings_updates Start  
2023-10-16 08:56:39 [info] Elementor/Upgrades - _v_3_17_0_site_settings_updates Finished 
2023-10-16 08:56:39 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.16.4',
  'to' => '3.16.5',
)]
2023-10-16 08:56:39 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.16.4',
  'to' => '3.16.5',
)]
2023-10-18 11:31:36 [info] elementor::elementor_updater Started 
2023-10-18 11:31:36 [info] Elementor/Upgrades - _on_each_version Start  
2023-10-18 11:31:36 [info] Elementor/Upgrades - _on_each_version Finished 
2023-10-18 11:31:36 [info] Elementor/Upgrades - _v_3_17_0_site_settings_updates Start  
2023-10-18 11:31:36 [info] Elementor/Upgrades - _v_3_17_0_site_settings_updates Finished 
2023-10-18 11:31:36 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.16.5',
  'to' => '3.16.6',
)]
2023-10-18 11:31:36 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.16.5',
  'to' => '3.16.6',
)]
2023-10-25 15:06:02 [info] elementor::elementor_updater Started 
2023-10-25 15:06:02 [info] Elementor/Upgrades - _on_each_version Start  
2023-10-25 15:06:02 [info] Elementor/Upgrades - _on_each_version Finished 
2023-10-25 15:06:02 [info] Elementor/Upgrades - _v_3_17_0_site_settings_updates Start  
2023-10-25 15:06:02 [info] Elementor/Upgrades - _v_3_17_0_site_settings_updates Finished 
2023-10-25 15:06:02 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.16.6',
  'to' => '3.17.0',
)]
2023-10-25 15:06:02 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.16.6',
  'to' => '3.17.0',
)]

JS: showing 7 of 7JS: 2023-09-25 11:49:56 [error X 7][https://xxxxx.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.16.4:3:567751] Blocked a frame with origin "https://xxxxx.com" from accessing a cross-origin frame. 
JS: 2023-09-25 12:09:18 [error X 2][https://xxxxx.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.16.4:3:567751] Blocked a frame with origin "https://xxxxx.com" from accessing a cross-origin frame. 
JS: 2023-10-25 13:49:10 [error X 2][https://xxxxx.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.17.0:3:669709] elementorFrontend.elements.window.jQuery is not a function 
JS: 2023-10-25 13:49:11 [error X 1][https://xxxxx.com/wp-includes/js/jquery/jquery.min.js?ver=3.7.0:2:85515] Cannot read properties of null (reading 'clientHeight') 
JS: 2023-10-25 13:49:12 [error X 1][https://xxxxx.com/wp-content/plugins/elementor/assets/js/frontend-modules.min.js?ver=3.17.0:2:16880] Cannot read properties of undefined (reading 'attributes') 
JS: 2023-10-25 13:57:57 [error X 2][https://xxxxx.com/wp-admin/load-scripts.php?c=0&loadchunk_0=jquery-core,jquery-migrate,utils&ver=6.3.2:2:28722] Cannot read properties of undefined (reading 'value') 
JS: 2023-10-25 13:58:22 [error X 1][https://xxxxx.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.17.0:3:805504] Cannot read properties of undefined (reading 'toLowerCase') 



== Elementor - Compatibility Tag ==
	
	Connect Polylang for Elementor: Incompatible
	Elementor Pro: Incompatible

== Elementor Pro - Compatibility Tag ==
	
	Connect Polylang for Elementor: Incompatible
```

**Comment by nicholaszein** [comment]
Hello!

This issue was solved with the release of Elementor Core v3.17.1.

Feel free to update to the latest version.

Kind regards
